packages/puppeteer-core/src/bidi/Deserializer.ts

```diff
@@ -27,25 +27,30 @@ export class BidiDeserializer {
     }
   }
 
-  static deserializeLocalValue(result: Bidi.Script.RemoteValue): unknown {
+  static deserializeRemoteValue(result: Bidi.Script.RemoteValue): any {
```

Comment:
```suggestion
  static deserialize(result: Bidi.Script.RemoteValue): any {
```

Reply:
I changed it to deserializeRemoteValue to mirror Serializer.

Reply:
I think we should keep simple names, as now we are duplicating the type in the function name without having another function. So we should change both places.